pkg/framework/plugins/nodeutilization/highnodeutilization.go

```diff
@@ -63,6 +64,7 @@ func NewHighNodeUtilization(
 			genericArgs,
 		)
 	}
+	logger := klog.Background().WithValues("plugin", HighNodeUtilizationPluginName)
```

Comment:
Would you please update https://github.com/kubernetes-sigs/descheduler/blob/e283c3103026a3d134cc86a0c0fd67a108674e31/pkg/framework/profile/profile.go#L369 instead? This will remove the duplication of setting the plugin name for each plugin explicitly. The same for https://github.com/kubernetes-sigs/descheduler/blob/e283c3103026a3d134cc86a0c0fd67a108674e31/pkg/framework/profile/profile.go#L340.